Docker/ods-api-db-admin/ubuntu/mssql/README.md

```diff
@@ -7,8 +7,9 @@ started for the first time.
 SA user is disabled after initial setup.
 
 > [!NOTE]
-> This image does not include any pre-installed vendors and is suitable for
-> [production use](#sql-server-2022-express-license-overview).
+> This image does not include any pre-installed vendors and running as a container on an Ubuntu 16.04 host (as per the
```

Comment:
The host should be Ubuntu 22.04 (instead of 16.04) because that's the version that the base image uses.